example/probe-demo/probe-demo.cpp

```diff
@@ -199,14 +199,14 @@ void vector_sampler(arb::probe_metadata pm, std::size_t n, const arb::sample_rec
 
         for (unsigned j = 0; j<n_entities; ++j) {
             std::cout << samples[i].time << ", ";
-			if (cables_ptr) {
-				arb::mcable where = (*cables_ptr)[j];
-				std::cout << where.prox_pos << ", " << where.dist_pos << ", ";
-			} else {
-				arb::mlocation loc = (*point_info_ptr)[j].loc;
-				std::cout << loc.pos << ", ";
-			}
-			std::cout << lo[j] << '\n';
+         if (cables_ptr) {
+            arb::mcable where = (*cables_ptr)[j];
+            std::cout << where.prox_pos << ", " << where.dist_pos << ", ";
+         } else {
+            arb::mlocation loc = (*point_info_ptr)[j].loc;
+            std::cout << loc.pos << ", ";
+         }
+         std::cout << lo[j] << '\n';
```

Comment:
```suggestion
             if (cables_ptr) {
                arb::mcable where = (*cables_ptr)[j];
                std::cout << where.prox_pos << ", " << where.dist_pos << ", ";
             } else {
                arb::mlocation loc = (*point_info_ptr)[j].loc;
                std::cout << loc.pos << ", ";
             }
             std::cout << lo[j] << '\n';
```

Reply:
That I don't understand?

Reply:
looks to me this part needs one more indentation, does it not?

Reply:
actually it only needs 3 additional spaces

Reply:
Ok, then it's the missing context!

Reply:
3 spaces ... that's weird, we should have 4n space indents.

Reply:
Fixed by hitting \t